src/server/multi_command_squasher.cc

```diff
@@ -87,10 +93,16 @@ void MultiCommandSquasher::ExecuteStandalone(StoredCmd* cmd) {
   DCHECK(order_.empty());  // check no squashed chain is interrupted
 
   auto* tx = cntx_->transaction;
-  tx->MultiSwitchCmd(cmd->descr);
-  if (cmd->descr->IsTransactional())
-    tx->InitByArgs(cntx_->conn_state.db_index, cmd->ArgList());
-  cmd->descr->Invoke(cmd->ArgList(), cntx_);
+  tx->MultiSwitchCmd(cmd->Cid());
+
+  char namebuf[32];
```

Comment:
Ok, how about we stop passing the command name via args?
Make a preliminary PR that removes the command and passes only the rest of the argument.